setup.js

```diff
@@ -70,7 +73,5 @@ async function createTables(dynamoDB, tables) {
 }
 
 async function deleteTables(dynamoDB, tableNames) {
-  return Promise.all(
-    tableNames.map(tableName => dynamoDB.deleteTable({TableName: tableName}).promise())
```

Comment:
Hey! Thank you for your contribution.

Why do you want to remove .promise() here?

Reply:
@randomhash Thanks for reviewing! `.promise()` failed for me because `jest-dynamodb` uses AWS SDK v3 but `.promise()` is only part of the AWS SDK v2 API.

`.promise()` is required with v2 because [deleteTable v2](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#deleteTable-property) returns an `AWS.Request` with a `promise` method that must be called. Instead, in v3 [deleteTable](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-dynamodb/classes/dynamodb.html#deletetable) returns a `Promise`.

Also see the [Migrating to AWS SDK v3 guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrating-to-v3.html#path2-examples) which includes an example of removing `.promise()` calls when upgrading to AWS SDK v3.